sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat/chat_completion_with_model_as_platform.ipynb

```diff
@@ -0,0 +1,565 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FineTuning LLM with Model-As-Service\n",
+    "\n",
+    "This sample shows how use create a standalone FineTuning job to fine tune a model to summarize a dialog between 2 people using samsum dataset.\n",
+    "\n",
+    "#### Training data\n",
+    "We use sample data placed along with the notebook for our finetuning in files \"train.jsonl\" and \"validation.jsonl\"\n",
+    "\n",
+    "#### Model\n",
+    "We will use the Phi-3-mini-4k-instruct model to show how user can finetune a model for chat-completion task. If you opened this notebook from a specific model card, remember to replace the specific model name. \n",
+    "\n",
+    "#### Outline\n",
+    "1. Setup pre-requisites\n",
+    "2. Pick a model to fine-tune.\n",
+    "3. Create training and validation datasets.\n",
+    "4. Configure the fine tuning job.\n",
+    "5. Submit the fine tuning job.\n",
+    "6. Create online endpoint deployment using finetuned model and sample inference."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Setup pre-requisites\n",
+    "* Install dependencies\n",
+    "* Connect to AzureML Workspace. Learn more at [set up SDK authentication](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk). Replace  `<WORKSPACE_NAME>`, `<RESOURCE_GROUP>` and `<SUBSCRIPTION_ID>` below.\n",
+    "* Connect to `azureml` system registry\n",
+    "* Set an optional experiment name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Install dependencies by running below cell. This is not an optional step if running in a new environment.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%pip install https://finetuningsdkteststorage.blob.core.windows.net/finetuning-private-sdk/azure_ai_ml-1.23.0-py3-none-any.whl\n",
```

Comment:
could we move the private whl to another storage account where we are getting exception for disabling anonymous access. lets not introduce new exceptions.